RobotCode/src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -14,8 +14,9 @@ public class IntakeSubsystem extends SubsystemBase
     // Intake toggle motor turns on or off the motors that actually take in notes (game pieces).
     // Internally, it is known as INTAKE_ON_MOTOR, but I renamed it for the sake of clarity.
     // The right motor is the main motor, and the left motor is the follow motor.
-    private final TeamSparkMAX INTAKE_RAISER_MOTOR_LEFT, INTAKE_RAISER_MOTOR_RIGHT,
-            INTAKE_TOGGLE_MOTOR;
+    private final TeamSparkMAX INTAKE_RAISER_MOTOR_LEFT;
```

Comment:
Convert comment to javadoc. Also move each line to its respective motor. Motors are named wrong; they are objects, not primitives, so should be FirstwordSecondWord.